src/passes/literalExpressionEvaluator/literalExpressionEvaluator.ts

```diff
@@ -236,6 +226,14 @@ function evaluateBinaryLiteral(node: BinaryOperation): RationalLiteral | boolean
         return left.shiftLeft(right);
       case '>>':
         return left.shiftRight(right);
+      case '&':
+        return left.and(right);
+      case '|':
+        return left.or(right);
+      case '^':
+        return left.xor(right);
+      case '~':
+        return left.not();
```

Comment:
This operator can be removed from this switch because `~` is a unary operator, and this is for binary operators